apps/keyboard/js/views/candidate_panel_view.js

```diff
@@ -0,0 +1,249 @@
+'use strict';
+
+/* global BaseView */
+
+(function(exports) {
+
+/**
+ * CandidateView handles the rendering of each candidate, i.e.
+ * the word suggestion.
+ */
+function CandidateView() {
+  BaseView.apply(this, arguments);
+}
+
+CandidateView.prototype = Object.create(BaseView.prototype);
+
+CandidateView.prototype.render = function render() {
+  var span = document.createElement('span');
+  span.textContent = this.target.text;
+  span.style.flex = this.options.unit;
+  // ui/integration test needs this
+  span.dataset.data = this.target.data;
+
+  this.element = span;
+
+  if (this.viewManager) {
```

Comment:
It's not. For the tests you could simply pass a `viewManagerStub`, which is really not hard to create.

Hint: `this.sinon.stub(Object.create(ViewManager.prototype));`. But that won't work for now because ViewManager is currently the IMERender, so you would have to create an object in place.